inst/bin/switch-phpfarm

```diff
@@ -5,21 +5,35 @@ basedir=`pwd`
 binlink="$basedir/current"
 active=`basename $(readlink "$binlink") 2>/dev/null | sed -e 's/^php-//;s/\./\\\\./g'`
 
+php_system=""
+if [ -f "/usr/bin/php" ]; then
+    php_system=$(/usr/bin/php -v|grep --only-matching --perl-regexp "\d\.\\d+\.\\d+"|head -1)
+fi
+
 if [ "$1" = "" ]; then
     echo Switch the currently active PHP version of phpfarm
     echo Available versions:
+    if [ "$php_system" != "" ]; then
+        echo "$php_system (default system version)" | sed -e 's/^/  /' -e "s/^  \($active .*\)\$/* \1/"
+    fi
     find . -maxdepth 1 -type d -name 'php-*' -printf '%f\n' | sort -V | sed -e 's/^php-//' -e 's/^/  /' -e "s/^  \($active\)\$/* \1/"
     exit 1
 fi
 
 version="$1"
 phpdir="php-$version"
 
-if [ ! -d "$basedir/$phpdir" ]; then
-    echo "PHP version $version is not installed"
-    exit 2
-fi
-
-echo Setting active PHP version to $version
-ln -sfT "$phpdir" "$binlink"
-$basedir/current/bin/php --version
+if [ $version != $php_system ]; then
+    if [ ! -d "$basedir/$phpdir" ]; then
+       echo "PHP version $version is not installed"
+       exit 2
+    else
+       echo Setting active PHP version to $version
+       ln -sfT "$phpdir" "$binlink"
+       $basedir/current/bin/php --version
+    fi
+else 
+   echo Setting active PHP version to system version ($version)
+   rm $binlink
```

Comment:
add -f so that no prompt is printed in case rm has been aliased to "rm -i"

also, you should put quotes around $binlink in case the path contains spaces